OpenTibia/Assets/Scripts/Modules/ToggleButtons/ToggleButtonsWidget.cs

```diff
@@ -161,7 +161,27 @@ public class ToggleButtonsWidget : UI.Legacy.SidebarWidget
         }
 
         public void OnLogoutButtonClick() {
+            var gameManager = OpenTibiaUnity.GameManager;
+
+            if (gameManager.IsGameRunning)
+            {
+                void OnLogoutPopupYesClick()
+                {
+                    if (gameManager.IsGameRunning)
```

Comment:
This is redundant, I think - the same check is performed on the line 166

Reply:
You are right.